plugin.audio.rco/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.rco" name="Het Koninklijk Concertgebouworkest" version="1.0.1" provider-name="bosel">
+  <requires>
+      <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
please set `xbmc.python` to 2.24.0 so it matches jarvis. See: http://kodi.wiki/view/addon.xml